accounts-db/src/accounts_db.rs

```diff
@@ -8870,9 +8871,30 @@ impl AccountsDb {
                             other.accounts_data_len_from_duplicates;
                         self.num_duplicate_accounts += other.num_duplicate_accounts;
                         self.uncleaned_roots.extend(other.uncleaned_roots);
-                        self.duplicates_lt_hash
-                            .0
-                            .mix_in(&other.duplicates_lt_hash.0);
+
+                        match (
+                            self.duplicates_lt_hash.is_some(),
```

Comment:
How about make a type for `Option<Box<DuplicatesLtHash>>` and extract this implement a `mix_in` method on the type?

Reply:
Yes, option, box they are important implementation details. 

If we make they a type and extract the details into a mix_in method, then we can unittest it. 

With it current form, it is very hard to unittest this piece of logic.

Reply:
That being said, I am sure that your code logic is right. 
Anyway, just a suggestion. I am fine with what it is now too.